ensembl/htdocs/info/genome/variation/species/sources_documentation.html

```diff
@@ -567,7 +567,7 @@ <h2 id="homo_sapiens" style="display:inline;color:#333">Human<span class="small
         
         <td style="width:4px;padding:0px;margin:0px;border-top:1px solid #FFF;border-bottom:1px solid #FFF;background-color:#090"></td>
         <td style="font-weight:bold"><a href="https://www.ncbi.nlm.nih.gov/clinvar/" style="text-decoration:none" target="_blank">ClinVar</a></td>
-        <td>08/2020</td>
+        <td>08/2020 09/2020</td>
```

Comment:
@ima23 Sorry, I don't know what this means, but just checking — is this correct? Two dates in the same table cell?

Reply:
@azangru , thanks for checking. Yes, for this release only the ClinVar version will have both the dates.